routeswitch.js

```
@@ -216,20 +215,20 @@ function loadHandlers (loader, path, log) {
         handlerStats.forEach(function(stat) {
             var handlerPath = Path.resolve(stat.name);
             try {
-                var handler = loader(handlerPath);
+                var handler = options.loader(handlerPath);
                 handlers.push(handler);
             } catch (e) {
                 if (stat.isDirectory()) {
                     // Try to recurse
                     subDirs.push(handlerPath);
-                } else if (log) {
-                    log('error/handler', e, stat.name, e && e.stack);
+                } else {
+                    throw e;
```

This feels ugly, but it doesn't seem like we want to throw any exceptions caused when `stat` is a directory, so we drop that case on the floor and retry, and otherwise re-throw the exception.